packages/visx-tooltip/test/Portal.test.tsx

```diff
@@ -1,7 +1,53 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { render, screen, within } from '@testing-library/react';
 import { Portal } from '../src';
 
+const PortalWithContainer = () => {
+  // useState rather than useRef so it will react to the ref being available for testing purposes
+  const [portalContainer, setPortalContainer] = React.useState<HTMLDivElement | null>(null);
+  const onRefChange = React.useCallback((node) => {
+    setPortalContainer(node);
+  }, []);
+  return (
+    <>
+      <div data-testid="inner-div" ref={onRefChange} />
+      {portalContainer && (
+        <Portal container={portalContainer}>
+          <div data-testid="element-in-portal" />
+        </Portal>
+      )}
+    </>
+  );
+};
+
 describe('Portal', () => {
   test('it should be defined', () => {
     expect(Portal).toBeDefined();
   });
+
+  it('should render children at the document root', async () => {
+    render(
+      <>
```

Comment:
would this make more sense as a test setup since you wouldn't expect to find `element-in-portal` inside `inner-div` if it's a sibling?
```suggestion
      <div data-testid="inner-div">
        <Portal>
          <div data-testid="element-in-portal" />
        </Portal>
      </div>
```

Reply:
That's a great point!